src/main/java/uk/gov/companieshouse/scanupondemand/orders/api/model/ScanUponDemandItem.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.companieshouse.scanupondemand.orders.api.model;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+/**
+ * An instance of this represents a scan upon demand item.
+ */
+
+@Document(collection = "scan-upon-demands")
+public class ScanUponDemandItem extends Item {
```

Comment:
You probably don't need the item class, you could put all of it in this file.

Reply:
Probably yes but I followed the certificates api example which includes the Item class. What is the best approach in this case as two APIs are implemented in different ways (in some parts)

Reply:
I would follow the Certified Copies API which just has CertifiedCopyItem and no Item class. The reason we have an Item class in the Certificates API is because originally we were going to have Certificates, Certified Copies and SCUD in one API, so we created a generic Item API class. We later decided to separate these APIs out so we no longer need the Item class. Ideally we should remove it from Certificates API as well.